cluster/ci/monitoring/mixins/prometheus/ghproxy.libsonnet

```diff
@@ -0,0 +1,45 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'ghproxy',
+        rules: [
+          {
+            alert: 'ghproxy-status-code-abnormal-%s' % std.strReplace(code, '..', 'XX'),
+            // excluding 404 because otherwise
+            // "this is going to spam us just about 24/7 with the OWNERS thing"
+            expr: |||
+              sum(rate(github_request_duration_count{status=~"%s", status!="404"}[5m])) > 0
+            ||| % code,
+            'for': '1m',
+            labels: {
+              severity: 'slack',
+            },
+            annotations: {
+              message: 'ghproxy has a increasing number of status code %s.' % std.strReplace(code, '..', 'XX'),
+            },
+          }
+          for code in ['4..', '5..']
+        ] +
+        [
+          {
+            alert: 'ghproxy-running-out-github-tokens-in-a-hour',
+            // here we check 1/3 of the capacity (1 hour): 1200s
+            expr: |||
+              github_token_reset <  1200 * 1000000000
```

Comment:
We need to make sure we are looking at the same `token_hash` fo this and for the usage

Reply:
I need more hint on this one.
I will take each `token_hash` into account.